docs/rest_api.md

```diff
@@ -1198,6 +1198,26 @@ Authorization requirements:
 
 * Administrative access.
 
+#### user_set_admin
+
+'PUT /auth/basic/user/<user>/set_admin'
```

Comment:
Maybe change the path component to `is_admin`; the general notion with REST Apis is that URLs represent objects, not functions. PUT /my-url` is conceptually the same as an assignment statement, so the URL should denote the "variable" being set.

I'm actually with @knikolla re: changing it to a `PATCH` on the user URL, though I don't feel that strongly. but the request body would make more sense that way.